### PR TITLE
Guide on using Intrinsics to Undistort

### PR DESCRIPTION
**Pull request by marc-tonsen**

### CODE REVIEW

File: src/invisible/how-tos/advanced-analysis/intrinsics/README.ipynb [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
...gQnSThK1sTZ0FzKAiBnTwJr5G1SSL0IlvukgAS4yKr5NNDNQBwsS90UeM3fUGCaS1LeAQMyzAzNeNAmmgLl82CWOqYFouMohtoro6sOv2fMV8xv1jo8Z0JQgSrsdV/zQAOvG5fNG8c3xWC18m1EbY6YyfaC0gawgZgwDIjjBEhjzWZoc07qgExjq26emZoLI9C5h4ZaQtDGd+MwYBoG3N/vlZMoaKfq3W7CPE2gEHBZi7KLBgAlIbB+fxwHiJEUnk4nPD4+4suXZ5xPi4KPuz3u7w748YePmOexAhFAw1tPT084vhwhWTBEbQY3jSOWyxkx7HB3uAOTgqxpmiqYAvRZf/38K06nE9Y1YZh2oKDNJb2lQIvkuGJu87g1Rtpe5Qp+fH4YEKqGjFQAcA0EUM/HgTEFvb7yw+ieFhEl8hsHJdv03KcKmnrPj9/ndlmyuVPIV4IIkK3qk6MiUaZaWuvI3MMPta/wDYzy39xxjdW61wH8TfHMZsirsaFqPFi4Hqx5UN4CJ9wFTJMaGT98+ojnF610+vL1EcfTCUnUYIzB4hMWetY6nIAQDay4IarN75Cz6lEHQlUUmt5QbKwJwFQKyL3DgeFkncSMIJoeomC9hbU3uVlEBl7UGKLAm9C3+vpant80TZjmSZvFsofAvz0x3wgtZdsKr92KzIwYAniKWAagHDOmYn1hSkG5FO3BIgWEgmCVRDV5tbrAuslmS1qClkvCXVFZs6E1GlVQJKmS8zBEIOzmEaPllVzWFS8vR6QlmdtZrOGjVlRpJVHQrteBEQPUFbcRaPaTSb0sJFopJYLBwiOATYgIEM2M8wXRLdpqdQtr0ph15k3rBf/8p3/E83rB/2O/w/hjBFFBZGA/q4t9YfNKFS3DjgxMYPD5jMs//Qm5ENbjgng8qfC08mU4ey3UQxYkIZaAOUSMwft0ZAQUSNEck1QyUtKU2WLhL4LmPWh1FrSKKIs24CRLQpMWNispK5qnFUxKdLbbDXD0rxn1gilGXEJAyQkQy2FofnEFhlWiexK2oXKS6vUW56WpSWbNQ9FgGFDFhnsv6it2ns6b8VuOrQ19/WZ7p7Fx2vOECC8iYIImNceoneSnHUIcwNAwSiwFGUp46BYTkSfWasm830hLWAfGcWdVExqDPxzulIMlRPPIuLfCPZSa0AowkgjY3ACn4xnrekGM2tYAVkY/T
```
Comment (marc-tonsen) [inline src/invisible/how-tos/advanced-analysis/intrinsics/README.ipynb:238]: @dourvaris I copied this code from an example you created on how to copy an audio stream over properly. When adding line 238 I get an error though:
`ValueError: Can only set extradata for decoders.`

Seems to run fine without this line though. Can you confirm that we can just drop it?

Comment (dourvaris) [reply src/invisible/how-tos/advanced-analysis/intrinsics/README.ipynb:238]: Yes

Comment (papr) [inline src/invisible/how-tos/advanced-analysis/intrinsics/README.ipynb:238]:
```suggestion
```

**Review by papr — APPROVED** [review]

**Comment by papr** [comment]
@marc-tonsen ready to merge from my POV. Feel free to hit the merge button when you see fit.

**Pull request merged**